src/exception/exception-api.ts

```diff
@@ -41,39 +37,37 @@ function getResStatus(result: exceptionService.Result): number {
   }
 }
 
+const validateProgramExceptionRecords = (records: ReadonlyArray<TsvUtils.TsvRecordAsJsonObj>) => {
+  if (!isReadonlyArrayOf(records, isProgramExceptionRecord)) {
+    L.debug(`Program Exception TSV_PARSING_FAILED`);
+    throw new Errors.TSVParseError();
+  }
+  return records;
+};
+
+const validateEntityExceptionRecords = (records: ReadonlyArray<TsvUtils.TsvRecordAsJsonObj>) => {
```

Comment:
To help future devs understand what this is doing, let's explicitly declare the return type:

```suggestion
const validateEntityExceptionRecords = (records: ReadonlyArray<TsvUtils.TsvRecordAsJsonObj>): ReadonlyArray<EntityExceptionRecord> => {
```

Reply:
Same for `validateProgramExceptionRecords` on line 40